<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>312</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>578</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>578</width>
      <height>312</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.324, y1:0.0975, x2:0.909091, y2:0.995, stop:0.130682 rgba(255, 0, 0, 255), stop:0.892045 rgba(52, 0, 0, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semibold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0.324, y1:0.0975, x2:0.852, y2:0.461, stop:0.130682 rgba(255, 0, 0, 0), stop:0.892045 rgba(52, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>Usuario </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semibold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0.324, y1:0.0975, x2:0.852, y2:0.461, stop:0.130682 rgba(255, 0, 0, 0), stop:0.892045 rgba(52, 0, 0, 0));</string>
     </property>
     <property name="text">
      <string>Senha</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0.324, y1:0.0975, x2:0.852, y2:0.461, stop:0.130682 rgba(255, 0, 0, 0), stop:0.892045 rgba(52, 0, 0, 0));</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0.324, y1:0.0975, x2:0.852, y2:0.461, stop:0.130682 rgba(255, 0, 0, 0), stop:0.892045 rgba(52, 0, 0, 0));</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>454</x>
       <y>202</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	
	background-color: rgb(181, 0, 0);
	color: rgb(255, 255, 255);
	border-radius: 15px;
}
QPushButton:hover{
	background-color: rgb(255, 0, 0);
	color: rgb(255, 255, 255);
	border-radius: 15px;
}
QPushButton:pressed{
	background-color: rgb(255, 255, 255);
	color: rgb(255, 255, 255);
	border-radius: 15px;
}</string>
     </property>
     <property name="text">
      <string>Entrar</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>291</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0.324, y1:0.0975, x2:0.852, y2:0.461, stop:0.130682 rgba(255, 0, 0, 0), stop:0.892045 rgba(52, 0, 0, 0));
image: url(:/ar/logofrutariacapixaba.webp);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagesn_arquivo_caixa.qrc"/>
 </resources>
 <connections/>
</ui>
